src/styles/blocks/reset.scss

```diff
@@ -0,0 +1,12 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
```

Comment:
It would be great to use code from the reset.css library or add reset to html in <link>